### PR TITLE
docs: Update README to match current status.

### PR DESCRIPTION
**Pull request by davecgh**
This consists of a series of commits that update the main `README.md` to match the current status and reality as well as address a few other things.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -228,9 +228,9 @@ provides a detailed example for reference.
 ## Pool Fees
 
 In mining pool mode pool fees collected by the pool operator are for maintaining
-a connection to the decred network for the delivery of work to mining clients,
-the submission of solved work by mining clients to the network and processing of
-block rewards based on work contributed by participting accounts.
+a connection to the decred network, for the delivery of work to mining clients,
```
Comment (davecgh) [inline README.md:231]:
```suggestion
a connection to the decred network, the delivery of work to mining clients,
````

Comment (davecgh) [reply README.md]: Done.

File: README.md [hunk 2]
```diff
@@ -159,9 +159,9 @@ dcrpool --configfile=pool.conf --appdata=/tmp/dcrpool-harness/pool
 ```no-highlight
 rpcuser=user
 rpcpass=pass
-dcrdrpchost=127.0.0.1:19556
+dcrdrpchost=127.0.0.1:9109
 dcrdrpccert=/home/.dcrd/rpc.cert
-walletgrpchost=127.0.0.1:19558
+walletgrpchost=127.0.0.1:9111
```
Comment (jholdstock) [inline README.md:162]: To remove a potential point of confusion, update `activenet` to `mainnet` so it matches the ports and the example above.

Comment (davecgh) [reply README.md:162]: Done.

**Review by jholdstock — APPROVED** [review]

**Pull request merged**